src/routes/analytics/DashboardPage.tsx

```diff
@@ -10,15 +10,12 @@ interface Props {
 export function DashboardPage(props: Props) {
   const dashboardId = parseInt(props.id, 10)
 
+  // When the site changed, reload to apply the site's sandboxed data.
+  useSiteChanged(window.location.reload)
+
   return (
     <Box mih="100vh" className="dashboard-container smartscalar">
-      <RemountOnSiteChange>
```

Comment:
Using `RemountOnSiteChange` doesn't actually work here - we still get stale data from the old user.